### PR TITLE
Add ctrl z `wait` command and add escape code `[2004h` `[2004l` filter in helpers

### PR DESCRIPTION
**Pull request by kochania**
## moler 3.5.0
* Unix commands - optional ctrl + z and kill on timeout - add `wait` call after `kill`
* Add helper for excape codes `[2004l` `[2004h`

### CODE REVIEW

**Comment by kochania** [comment]
Issue with exiting to UNIX_LOCAL ("Stopped jobs")
[moler_log_snippet.log](https://github.com/nokia/moler/files/14932686/moler_log_snippet.log)

**Review by tkrol88 — APPROVED** [review]

File: moler/cmd/unix/genericunix.py [hunk 1]
```diff
@@ -126,22 +127,36 @@ def is_end_of_cmd_output(self, line: str) -> bool:
...
         return super(GenericUnixCommand, self).is_end_of_cmd_output(line=line)
 
     # [2]+  Stopped
     _re_ctrl_z_stopped = re.compile(r"\[(?P<JOB_ID>\d+)\]\+\s+Stopped")
 
+    # [2]+  Done
+    _re_kill_done = re.compile(r"\[(?P<JOB_ID>\d+)\]\+\s+Done")
+
+    # -bash: wait: %2: no such job
+    _re_kill_no_job = re.compile(r":\s\%(?P<JOB_ID>\d+): no such job")
```
Comment (marcin-usielski) [inline moler/cmd/unix/genericunix.py:141]: Do you use group JOB_ID from this regex in any place?

File: moler/cmd/unix/genericunix.py [hunk 2]
```diff
@@ -126,22 +127,36 @@ def is_end_of_cmd_output(self, line: str) -> bool:
...
-        if not self._kill_ctrl_z_sent and self._ctrl_z_sent:
+        if not self._kill_ctrl_z_job_done and self._ctrl_z_sent:
             return False
         return super(GenericUnixCommand, self).is_end_of_cmd_output(line=line)
 
     # [2]+  Stopped
     _re_ctrl_z_stopped = re.compile(r"\[(?P<JOB_ID>\d+)\]\+\s+Stopped")
 
+    # [2]+  Done
+    _re_kill_done = re.compile(r"\[(?P<JOB_ID>\d+)\]\+\s+Done")
```
Comment (marcin-usielski) [inline moler/cmd/unix/genericunix.py:138]: Do you use group JOB_ID from this regex in any place?

Comment (kochania) [reply moler/cmd/unix/genericunix.py]: You are right, group is unnecessary. Matching is a point here. Corrected.

**Pull request merged**